app/jobs/resolution_proofing_job.rb

```diff
@@ -79,8 +79,8 @@ def make_vendor_proofing_requests(
       threatmetrix_session_id: threatmetrix_session_id,
       request_ip: request_ip,
       should_proof_state_id: should_proof_state_id,
-      double_address_verification: double_address_verification,
       timer: timer,
+      double_address_verification: double_address_verification,
```

Comment:
It's often best to revert to the exact prior code when there's no functional change. This makes tracking down the source of issues with a file much easier.

Reply:
good point! i'll update it